README.md

```diff
@@ -1 +1,56 @@
-# scm-router
\ No newline at end of file
+# SCM Router
+[![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][status-image]][status-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
+
+> A generic scm plugin that routes builds to a specified scm
+
+A scm is interface for interacting with a source control management system.
+
+i.e. Github, Bitbucket, Gitlab
+
+The scm router will allow multiple scms to be used in a Screwdriver cluster.
+
+## Usage
+
+```bash
+npm install screwdriver-scm-router
+```
+
+### Interface
+
+It will initialize any routers specified in the [default.yaml](https://github.com/screwdriver-cd/screwdriver/blob/master/config/default.yaml#L123-L156) under the `scm` and `scms` keyword. At least one of `scm` and `scms` is required.
+
+**Example scm yaml section:**
+```
+scm:
+    plugin: github
+    github:
+        oauthClientId: YOU-PROBABLY-WANT-SOMETHING-HERE
+        ......
+scms:
```

Comment:
Looks like in the API it's actually `OR` case: https://github.com/s-yoshika/screwdriver/blob/b3f6b95398b63c29c6c4f81806bc685d6e3fac5e/bin/server#L54

So this needs to be changed.